tests/sentry/search/events/test_builder.py

```diff
@@ -1707,6 +1708,24 @@ def test_no_error_if_aggregates_disallowed_but_no_aggregates_included(self):
             use_aggregate_conditions=False,
         )
 
+    def test_multiple_dataset_but_no_data(self):
+        """When there's no data from the primary dataset we shouldn't error out"""
+        result = MetricsQueryBuilder(
+            self.params,
+            selected_columns=[
+                "p50()",
+                "count_unique(user)",
+            ],
+            allow_metric_aggregates=False,
+            use_aggregate_conditions=True,
+        ).run_query("test")
+        print(result["data"])
```

Comment:
```suggestion
```

Reply:
that was lazy :)

Reply:
;) could've saved me time by suggesting from the start